virtual-host-gatherer/lib/gatherer/modules/Xen.py

```diff
@@ -96,7 +96,7 @@ def run(self):
         try:
             protocol = 'https' if self.port == 443 else 'http'
             session = XenAPI.Session("%s://%s:%s" % (protocol, self.host, self.port))
-            session.xenapi.login_with_password(self.user, self.password, "2.3", "Xen Gatherer")
+            session.xenapi.login_with_password(self.user, self.password)
```

Comment:
Can we be sure that the library's output is stable or could the data parsing break when the API Version changes? Is there some kind of "we stay backwards compatible" policy we can rely on?

Hardcoding the version *might* be required if XenAPI reserves the right change its output.

Reply:
> Can we be sure that the library's output is stable or could the data parsing break when the API Version changes?

"Nothing's ever for sure, John. That's the only sure thing I do know." Charles Herman

> Is there some kind of "we stay backwards compatible" policy we can rely on?
> Hardcoding the version _might_ be required if XenAPI reserves the right change its output.

Thanks, that's a good point.  Unfortunately, I won't have more time to work on this.
Feel free to take ownership on it, or discard it or whatever it makes more sense.